index.js

```diff
@@ -29,22 +65,42 @@ server.post('/api/register', (req, res) => {
 })
 
 server.post('/api/login', (req, res) => {
+    // grab creds
     const creds = req.body;
 
+    // find the user
     db('users')
         .where({username: creds.username})
         .first()
         .then(user => {
+            // check creds
```

Comment:
We put our checks on line 76 because one we have a valid username, we can then check for the passoword against the hashed password